Mailer/MailerInterface.php

```diff
@@ -35,4 +35,22 @@ public function sendConfirmationEmailMessage(UserInterface $user);
      * @return void
      */
     public function sendResettingEmailMessage(UserInterface $user);
+
+    /**
+     * Define the email address used to send confirmation emails from.
+     *
+     * @param string|null $email
+     *
+     * @return void
+     */
+    public function setFromEmailConfirmation($email);
```

Comment:
This is totally impossible. It is a BC break and there is no reason to force everyone to implement such methods